src/main/java/core/mate/academy/service/MachineProducer.java

```diff
@@ -1,11 +1,8 @@
 package core.mate.academy.service;
 
+import core.mate.academy.model.Machine;
 import java.util.List;
 
-public interface MachineProducer {
-    /**
-     * In your implementations - create 2-3 models, add them to the list and return
-     * @return - the list of models
-     */
-    List<Object> get();
+public interface MachineProducer<T extends Machine> {
+    List<Machine> get();
```

Comment:
let's make this method return List of any Machine (`List<Excavator>` or `List<Truck>`)